### PR TITLE
LPS-133393 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133393
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 1]
```diff
@@ -403,4 +403,79 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-119582. Widget Search Result must shown content page and web content after execute reindex all on Search Page."
+	@priority = "5"
+	test ViewSearchResultOnPageAfterExecuteReindexAll {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:408]: ViewContentPageShownInSearchResultsAfterExecuteReindexAll

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 2]
```diff
@@ -403,4 +403,79 @@ definition {
...
+				title = "Searchable Web Content Title");
+		}
+
+		task ("Navigate to the search page") {
+			Navigator.gotoSitePage(
+				pageName = "Search",
+				siteName = "Test Site Name");
+		}
+
+		task ("Search by the token word") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:445]: Search keyword via search bar

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 3]
```diff
@@ -403,4 +403,79 @@ definition {
...
+			SearchAdministration.executeSearchActions(actionsDescription = "Reindex all search indexes.");
+		}
+
+		task ("Navigate to the search page") {
+			Navigator.gotoSitePage(
+				pageName = "Search",
+				siteName = "Test Site Name");
+		}
+
+		task ("Search by the token word") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:470]: Search keyword via search bar

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 4]
```diff
@@ -403,4 +403,79 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-119582. Widget Search Result must shown content page and web content after execute reindex all on Search Page."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:406]: View content page shown in search results after execute reindex all

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 5]
```diff
@@ -403,4 +403,79 @@ definition {
...
+		}
+
+		task ("Search by the token word") {
+			Search.searchCP(searchTerm = "Searchable");
+		}
+
+		task ("View both the content page and web content shown in the search results") {
+			Search.viewResultAssetTitle(searchAssetTitle = "Test Content Page Name");
+
+			Search.viewResultAssetTitle(searchAssetTitle = "Searchable Web Content Title");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:452]: Use loop to simplify case

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 6]
```diff
@@ -403,4 +403,79 @@ definition {
...
+		}
+
+		task ("Search by the token word") {
+			Search.searchCP(searchTerm = "Searchable");
+		}
+
+		task ("View both the content page and web content are still shown in the search results") {
+			Search.viewResultAssetTitle(searchAssetTitle = "Test Content Page Name");
+
+			Search.viewResultAssetTitle(searchAssetTitle = "Searchable Web Content Title");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:477]: Use loop to simplify case

**Comment by Tim-Cao** [comment]
Hi @ozysouza, Could you update the cases according to comments?

**Comment by ozysouza** [comment]
Hi @Tim-Cao PR updated following your comments.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 7]
```diff
@@ -403,4 +403,79 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-119582. View content page shown in search results after execute reindex all."
+	@priority = "5"
+	test ViewContentPageShownInSearchResultsAfterExecuteReindexAll {
+		property test.name.skip.portal.instance = "ContentPagesWithSearch#ViewSearchResultOnPageAfterExecuteReindexAll";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:409]: ContentPagesWithSearch#ViewContentPageShownInSearchResultsAfterExecuteReindexAll

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/ac4871830f25da0ca79a683af0206bb5a6aa67ff...ef0fb843e969d8006cc2f00640f71de780837f6c
in https://github.com/liferay-echo/liferay-portal/pull/5028

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**